recipes/srt/all/patches/001-cmake-link-targets.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 7fafe77cc..d5c4d701e 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -623,7 +623,7 @@ if (srt_libspec_shared)
+ 	set_target_properties (${TARGET_srt}_shared PROPERTIES VERSION ${SRT_VERSION} SOVERSION ${SRT_VERSION_MAJOR})
+ 	list (APPEND INSTALL_TARGETS ${TARGET_srt}_shared)
+ 	if (ENABLE_ENCRYPTION)
+-		target_link_libraries(${TARGET_srt}_shared PRIVATE ${SSL_LIBRARIES})
++		target_link_libraries(${TARGET_srt}_shared PRIVATE ${CONAN_LIBS_OPENSSL})
```

Comment:
If we are patching the original files, let's use the targets provided by the requirements

```suggestion
+		target_link_libraries(${TARGET_srt}_shared PRIVATE OpenSSL::OpenSSL)
```